### PR TITLE
LPS-166513 As a Site Admistrator, I want the exported Site and Portlet LAR files to be removed from the Document and Media library

### PR DESCRIPTION
**Pull request by hhtank**
@vendeltoreki 
This is the revised PR for LPS-166513. Things that were adjusted as discussed base on BChan review:
- Justify the debug log message.
- Keep the new-added configurations, but disable the cleanup process entirely by default (no scheduler job by default)

Please help see if this makes sense. Thanks,

### CODE REVIEW

**Comment by JoyceWang08211** [comment]
ci:test:bundle

**Comment by JoyceWang08211** [comment]
> ###  ci:test:staging - 186 out of 198 jobs passed in 1 hour 53 minutes
> Click here for more details.

No unique failures were found.

**Comment by JoyceWang08211** [comment]
ci:test:relevant

File: modules/apps/export-import/export-import-service/src/main/java/com/liferay/exportimport/internal/scheduler/DeleteExpiredBackgroundTasksSchedulerJobConfiguration.java [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+
+	@Override
+	public UnsafeRunnable<Exception> getJobExecutorUnsafeRunnable() {
+		return () -> _companyLocalService.forEachCompanyId(
+			companyId -> _deleteExpiredBackGroundTasks(companyId));
+	}
+
+	@Override
+	public TriggerConfiguration getTriggerConfiguration() {
+		return TriggerConfiguration.createTriggerConfiguration(
```
Comment (vendeltoreki) [inline modules/apps/export-import/export-import-service/src/main/java/com/liferay/exportimport/internal/scheduler/DeleteExpiredBackgroundTasksSchedulerJobConfiguration.java:69]: If the `cleanupJobInterval()` value is 0, then we get an exception at portal startup:
https://github.com/liferay/liferay-portal/blob/fa01b638b5bd41ad7057f3fb274a710c99444a3a/portal-kernel/src/com/liferay/portal/kernel/scheduler/TriggerConfiguration.java#L47-L50
I think this should be avoided somehow. I haven't found any examples yet, how can we disable scheduled jobs in this case.
If it's not possible, we can add a non-zero default value to the config, or we can even hardcode this value.
The instance-level configuration still disables the actual deletion even if this job runs regularly.

**Review by vendeltoreki — CHANGES_REQUESTED** [review]
Please fix the portal startup exception. Otherwise, the code looks good to me.

File: modules/apps/export-import/export-import-service/src/main/java/com/liferay/exportimport/internal/scheduler/DeleteExpiredBackgroundTasksSchedulerJobConfiguration.java (hunk 1 above)
Comment (hhtank) [reply modules/apps/export-import/export-import-service/src/main/java/com/liferay/exportimport/internal/scheduler/DeleteExpiredBackgroundTasksSchedulerJobConfiguration.java:69]: yeah, i found a sample in another app that catch the exception, let me see if that helps. thanks,

Comment (hhtank) [reply modules/apps/export-import/export-import-service/src/main/java/com/liferay/exportimport/internal/scheduler/DeleteExpiredBackgroundTasksSchedulerJobConfiguration.java:77]: > If the `cleanupJobInterval()` value is 0, then we get an exception at portal startup: https://github.com/liferay/liferay-portal/blob/fa01b638b5bd41ad7057f3fb274a710c99444a3a/portal-kernel/src/com/liferay/portal/kernel/scheduler/TriggerConfiguration.java#L47-L50 I think this should be avoided somehow. I haven't found any examples yet, how can we disable scheduled jobs in this case. If it's not possible, we can add a non-zero default value to the config, or we can even hardcode this value. The instance-level configuration still disables the actual deletion even if this job runs regularly.

Hi @vendeltoreki 
I wasn't able to disable the scheduled job when interval is 0. My workaround is hardcoding a far future scheduled date Dec 31 2099. Can you help check if that makes sense?
Thanks,

**Comment by hhtank** [comment]
ci:test:relevant
ci:test:sf
ci:test:staging

**Pull request closed without merging**